docs/notebooks/Similarity_Queries.ipynb

```diff
@@ -60,8 +60,11 @@
    "source": [
     "from gensim import corpora, models, similarities\n",
     "\n",
-    "dictionary = corpora.Dictionary.load('/tmp/deerwester.dict')\n",
-    "corpus = corpora.MmCorpus('/tmp/deerwester.mm') # comes from the first tutorial, \"From strings to vectors\"\n",
+    "try:\n",
+    "    dictionary = corpora.Dictionary.load('/tmp/deerwester.dict')\n",
+    "    corpus = corpora.MmCorpus('/tmp/deerwester.mm') # comes from the first tutorial, \"From strings to vectors\"\n",
+    "except:\n",
+    "    raise ValueError(\"SKIP: Dependency - Corpora_and_Vector_Spaces.ipynb file\")\n",
```

Comment:
Moved the Similarity queries tutorial to the Corpora_and_Vector_Spaces.ipynb file. 
Will change links within tutorials.md if this is alright.